CONTRIBUTING.md

```diff
@@ -1,7 +1,50 @@
 OpenProject is an open source project and we encourage you to help us out. We'd be happy if you do one of these things:
```

Comment:
I think you should also reword this intro or keep the list of things that someone might want to do to help us (create tickets, be active in forum, etc.).

**tl; dr:** Your write-up below does not match this paragraph